### PR TITLE
add title suffix for wpf and winforms

### PR DESCRIPTION
**Pull request by mairaw**
Issues like #10270 could be due to the lack of information identifying the technology

### CODE REVIEW

File: docfx.json [hunk 1]
```diff
@@ -436,6 +436,8 @@
         "_csharplang/proposals/csharp-7.3/*.md": "C# 7.3 specification proposals",
         "_csharplang/proposals/csharp-8.0/*.md": "C# 8.0 specification proposals",
         "docs/framework/wcf/**/**.md": "WCF",
+        "docs/framework/winforms/**/**.md": "Windows Forms",
+        "docs/framework/wpf/**/**.md": "WPF",
```
Comment (pkulikov) [inline docfx.json]: There is also a related `docs/desktop-wpf` folder. Should it be in this list as well or it's maintained in a different way?

Comment (mairaw) [reply docfx.json]: We're moving towards adding suffixes to all areas here but doing with different folders each time to be able to spot some oddities. @Thraka can you look at the desktop-wpf suffixes in a separate PR?

**Comment by mairaw** [comment]
Do you think we should remove the suffix from articles such as https://review.docs.microsoft.com/en-us/dotnet/framework/winforms/?branch=pr-en-us-16142 or https://review.docs.microsoft.com/en-us/dotnet/framework/winforms/windows-forms-overview?branch=pr-en-us-16142?

**Review by gewarren — COMMENTED** [review]
Left two suggestions. I also pulled your branch down and found a bunch of extra titles that should be modified. I was going to create a PR to your branch, but there are quite a lot and this PR would grow quite big, so should I just create a separate PR to master instead?

**Comment by gewarren** [comment]
> Do you think we should remove the suffix from articles such as https://review.docs.microsoft.com/en-us/dotnet/framework/winforms/?branch=pr-en-us-16142 or https://review.docs.microsoft.com/en-us/dotnet/framework/winforms/windows-forms-overview?branch=pr-en-us-16142?

Definitely.

**Comment by gewarren** [comment]
Here's my PR to your fork - https://github.com/mairaw/docs/pull/12

**Comment by mairaw** [comment]
Let's merge the PR you made on my fork and we can do separate ones directly to master.

**Comment by mairaw** [comment]
I think this is ready to merge @gewarren.

**Review by gewarren — APPROVED** [review]
LGTM!

**Review by Thraka — APPROVED** [review]
I like it.

**Pull request merged**